inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -31,7 +31,6 @@
 
 @hepnames.over('acquisition_source', '^541[10_].')
 @utils.for_each_value
-@utils.filter_values
```

Comment:
Apparently my commit message regarding this disappeared (it was in my first commit: https://github.com/tomaszgy/inspire-next/pull/1). We decided to do this because we will be filtering all None values later and this `filter_values` doesn't work with nested structures at all.